.circleci/config.yml

```diff
@@ -90,6 +90,15 @@ jobs:
             source emsdk_env.sh
             python scripts/test.py
 
+  build-docker-image:
+    machine: true
+    steps:
+      - checkout
+      - run: docker build -t emscripten-core/emsdk:$CIRCLE_BRANCH docker/
+      - run: |
+          echo "$DOCKER_PASS" | docker login --username $DOCKER_USER --password-stdin
```

Comment:
Is this the recommended way to authenticate the push ?  I wonder if we can pass login credentials directly to the push command somehow?

Reply:
This is from the CircleCI docs, yes. It is definitely possible to pass the password directly:

```
docker login -u "$DOCKER_USER" -p "$DOCKER_PASS" ...
```

Reply:
Pattern from the CircleCI docs, seems fine

Reply:
Looks odd, but also I've found that the recommended - more secure way for login.

Reply:
> Error: Cannot perform an interactive login from a non TTY device

Looks like `--password-stdin` is not accepted here anyways? 🤔